bftengine/src/bftengine/SimpleClientImp.cpp

```diff
@@ -426,39 +404,37 @@ class SeqNumberGeneratorForClientRequestsImp : public SeqNumberGeneratorForClien
   virtual uint64_t generateUniqueSequenceNumberForRequest() override;
 
  protected:
-  uint64_t lastMilliOfUniqueFetchID = 0;
-  uint32_t lastCountOfUniqueFetchID = 0;
+  uint64_t _lastMilliOfUniqueFetchID = 0;
+  uint32_t _lastCountOfUniqueFetchID = 0;
 };
 
 uint64_t SeqNumberGeneratorForClientRequestsImp::generateUniqueSequenceNumberForRequest() {
   std::chrono::time_point<std::chrono::system_clock> n = std::chrono::system_clock::now();
 
   uint64_t milli = std::chrono::duration_cast<std::chrono::milliseconds>(n.time_since_epoch()).count();
 
-  if (milli > lastMilliOfUniqueFetchID) {
-    lastMilliOfUniqueFetchID = milli;
-    lastCountOfUniqueFetchID = 0;
+  if (milli > _lastMilliOfUniqueFetchID) {
+    _lastMilliOfUniqueFetchID = milli;
+    _lastCountOfUniqueFetchID = 0;
   } else {
-    if (lastCountOfUniqueFetchID == 0x3FFFFF) {
+    if (_lastCountOfUniqueFetchID == 0x3FFFFF) {
       LOG_WARN(GL, "Client SeqNum Counter reached max value");
-      lastMilliOfUniqueFetchID++;
-      lastCountOfUniqueFetchID = 0;
+      _lastMilliOfUniqueFetchID++;
+      _lastCountOfUniqueFetchID = 0;
     } else {
-      lastCountOfUniqueFetchID++;
+      _lastCountOfUniqueFetchID++;
     }
   }
 
-  uint64_t r = (lastMilliOfUniqueFetchID << (64 - 42));
-  Assert(lastCountOfUniqueFetchID <= 0x3FFFFF);
-  r = r | ((uint64_t)lastCountOfUniqueFetchID);
+  uint64_t r = (_lastMilliOfUniqueFetchID << (64 - 42));
+  Assert(_lastCountOfUniqueFetchID <= 0x3FFFFF);
+  r = r | ((uint64_t)_lastCountOfUniqueFetchID);
 
   return r;
 }
 
 }  // namespace impl
-}  // namespace bftEngine
 
-namespace bftEngine {
```

Comment:
I don't believe moving this into the impl namespace is correct. The caller should never use the impl namespace directly. l think that's why they were separated.

Reply:
Oh, no I did not move it to impl namespace, just removed duplication. Take a look at a full file.

Reply:
Oh you are right! My bad.